docs/user-guide.md

```diff
@@ -124,9 +124,12 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Create the cluster configuration file by editing the `cluster_config.json` file template that is provided with the toolkit.
+1. Create the cluster configuration. You have two options:
+   1. edit the `cluster_config.json` JSON file template that is provided with the toolkit, 
+and supply its location as `--cluster-config` parameter
+   1. supply the cluster configuration JSON directly as `--cluster-config-json` parameter   
```

Comment:
"Pass the cluster configuration JSON as an argument to the `--cluster-config-json` parameter"